hack/update-testdata.sh

```diff
@@ -23,6 +23,12 @@ ROOT_DIR="$(realpath "${DIR}/..")"
 
 function update() {
   go test -v ./test/e2e/kwokctl/dryrun -args --update-testdata
+  go test -v ./test/e2e/kwokctl/binary -args --update-testdata
+  go test -v ./test/e2e/kwokctl/docker -args --update-testdata
+  go test -v ./test/e2e/kwokctl/kind -args --update-testdata
+  go test -v ./test/e2e/kwokctl/kind-podman -args --update-testdata
+  go test -v ./test/e2e/kwokctl/nerdctl -args --update-testdata
+  go test -v ./test/e2e/kwokctl/podman -args --update-testdata
```

Comment:
This seems to run all of kwokctl's tests, I'd like to know how it works because docker and podman can't work together on the same machine as far as I know.

Reply:
Is there any other way through which we can update the testdata for dryrun tests in all these environments?

Reply:
I think we can suggest in the error log which command can be used to update the testdata

Reply:
@wzshiming I have implemented the change suggested by you.Please take a look.